src/pages/og-image/ImageMeta.astro

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
Within Astro anything inside `/pages` becomes a route, for structure this component should be moved to `src/components`

Reply:
Ah good call, moved